scripts/electronpublish.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+const path = require('path');
+const mkdirp = require('mkdirp');
+const util = require('util');
+const rimraf = util.promisify(require('rimraf'));
+const archiver = require('archiver');
+const baseDir = path.join(__dirname, '..');
+const buildDir = path.join(baseDir, 'build');
+const distDir = path.join(baseDir, 'dist');
+
+(async () => {
+  await rimraf(distDir);
+  await mkdirp(distDir);
+  const folders = await fs.promises.readdir(buildDir);
+  const promises = folders.map((folder) => {
+    const source = path.join(buildDir, folder);
+    const destination = path.join(distDir, folder + '.zip');
```

Comment:
```suggestion
    const destination = path.join(distDir, `${folder}.zip`);
```

Reply:
I personally find that syntax harder to read when just concatenating two string.
But we already have that in the codebase so I will probably update it.